### DISCUSSION TITLE
txt files allow comments, but whether it is #xxxx or #xxxx#, I cannot remove the comment content when automatically completing the tag.

### DISCUSSION THREAD

**Discussion by 1025307336** [mcmonkeyprojects/SwarmUI#77] [Q&A]
If you want to have autocompletions for word lists (such as anime booru tags), you can! You just need to set it up first:

Find a word-list file. There are [several here you can use](https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/tree/main/tags).
Any will do (if the word is the first entry per row). Optionally second column can be an ID number 0-5 for unique colorations to distinguish categories..csv
or files (for newline-separated wordlists, allowing to mark comments)..txt#

How to use the # comment in the Autocompletions instructions?

**Comment by mcmonkey4eva** [comment]
If you want a comment you can
```
# some not used text
actualword
otherword
```
